### DISCUSSION TITLE
npx hardhat doesn't allow selection

### DISCUSSION THREAD

**Discussion by fadai88** [NomicFoundation/hardhat#3313] [General]
![image](https://user-images.githubusercontent.com/12747028/199004823-2dfe3328-11b0-4ee7-9069-3b43b35ee2cb.png)

When I run npx hardhat in the folder where I installed hardhat, it doesn't allow me to choose a Javascript project etc but runs as in the attached file.

My OS is Windows 10

**Comment by fvictorio** [comment]
This almost always means you have a `hardhat.config.js` file in an upper directory. Run `npx hardhat --verbose` and check the first messages; it should tell you which config file is detecting.

**Comment by Kriptocourier** [comment]
I'm having the same problem, checked the messages, and attached is a pic. I'm SUPER new to coding and trying to do a course, any idea what to do next so that I can start a new simple project?
![scrnsht10](https://user-images.githubusercontent.com/125918199/222596662-bc232959-22ab-4053-ab08-859dcc0be546.png)

**Reply by fvictorio** [reply]
You have a hardhat config in `/home/jon/code/hardhat.config.js`. Either remove it, or try creating the project in a directory that is not under that one.
